mono_repo/CHANGELOG.md

```diff
@@ -1,24 +1,47 @@
+## 3.1.0-beta.4
+
+### GitHub actions
+
+* Allow configuring different workflows for different stages.
+
+    ```yaml
+    github:
+      # You can group stages into individual workflows  
+      workflows:
+        # The key here is the name of the file - .github/workflows/lint.yml
+        lint:
+          # This populates `name` in the workflow
+          name: Dart Lint CI
+          # These are the stages that are populated in the workflow file
+          stages:
+          - analyze
+      # Any stages that are omitted here are put in a default workflow 
+      # named `dart.yml`.
+    ```
+
 ## 3.1.0-beta.3
 
-* GitHub actions:
-  * Change default branches to be `['master', 'main']`.
-    `$default-branch` is for templates.
+### GitHub actions:
+
+* Change default branches to be `['master', 'main']`.
+`$default-branch` is for templates.
 
 ## 3.1.0-beta.2
 
-* GitHub actions:
-  * **BREAK** from previous betas: `ci` is no longer supported as a top-level
-    value in `mono_repo.yaml`. Instead, the top-level keys `travis` and `github`
-    are used. If they exist (even if empty) they enable that CI target.
-  * Added `github` top-level key.
-    * Configure the `on` value to control what triggers the generated defined
-      workflow.
-    * A short-cut value `cron` is also supported – it can be a String that
-      adds a single scheduled job to the default `on` value.
-  * Support `stable` as a valid Dart SDK label.
-  * Use `[$default-branch]` instead of `[master]`.
-  * Prints the Dart SDK version after installation.
-  * Now supports `self_validate`.
+### GitHub actions
+
+* **BREAK** from previous betas: `ci` is no longer supported as a top-level
+value in `mono_repo.yaml`. Instead, the top-level keys `travis` and `github`
+are used. If they exist (even if empty) they enable that CI target.
+* Added `github` top-level key.
+* Configure the `on` value to control what triggers the generated defined
+  workflow.
+* A short-cut value `cron` is also supported – it can be a String that
+  adds a single scheduled job to the default `on` value.
```

Comment:
```suggestion
  * Configure the `on` value to control what triggers the generated defined
    workflow.
  * A short-cut value `cron` is also supported – it can be a String that
    adds a single scheduled job to the default `on` value.
```